autoload/vim_dadbod_completion.vim

```diff
@@ -163,7 +163,8 @@ function! s:generate_query(db, query_key, ...) abort
   if a:0 > 0
     let Query = Query(a:1)
   endif
-  return printf('%s %s', base_query, Query)
+  " for PostgreSQL
+  return printf('%s %s', substitute(base_query, '"', '', 'g'), Query)
```

Comment:
Why this has to be done? Isn't dadbod taking care of this?

Reply:
A string like the following was returned

```
psql -w --dbname "postgresql://postgres:postgres@127.0.0.1:5432/shop"
```

However, it was confirmed that it works normally even if you don't replace it with `substitute()`.
Therefore, it is removed.